src/main/java/edu/unh/cs753853/team1/QueryParagraphs.java

```diff
@@ -56,53 +60,44 @@ private void indexPara(IndexWriter iw, Data.Paragraph para) throws IOException {
 		Document paradoc = new Document();
 		paradoc.add(new StringField("paraid", para.getParaId(), Field.Store.YES));
 		paradoc.add(new TextField("parabody", para.getTextOnly(), Field.Store.YES));
+		FieldType indexType = new FieldType();
+		indexType.setIndexOptions(IndexOptions.DOCS_AND_FREQS);
+		indexType.setStored(true);
+		indexType.setStoreTermVectors(true);
+
+		paradoc.add(new Field("content", para.getTextOnly(), indexType));

```

Comment:
Add a new filed called "content". This filed store term vectors for all term in documents. It's for function anc.apc to get the max term freq from document.